docusaurus.config.js

```diff
@@ -415,4 +434,20 @@ const config = {
     }),
 };
 
+// Only for Testing purposes
```

Comment:
Since this is only for testing purpose, please delete it ;)

Reply:
I also think the dropdown should point to the current (released) not to the next? Same applies for the tiles on https://eclipse-tractusx.github.io/Kits